source/containers.tex

```diff
@@ -1520,6 +1516,10 @@
 comparison object
 \tcode{comp},
 \tcode{comp(k1, k2) == false \&\& comp(k2, k1) == false}.
+\begin{note}
+This is not necessarily the same as the result of the \tcode{operator==}
+on keys.
```

Comment:
Maybe "This is not necessarily the same as the result of `k1 == k2`." ?

Reply:
Fixed.